ajax/libs/choices.js/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "choices.js",
+  "version": "2.0.2",
+  "description": "A vanilla JS customisable text input/select box plugin",
+  "main": "./assets/scripts/dist/choices.min.js",

```

Comment:
@iv-craig Let's remove some fields we don't need. Like `main`, `scripts`, `bugs`, `devDependencies`.